src/main/java/com/double_o/dambap/s3/presentation/S3Controller.java

```diff
@@ -0,0 +1,56 @@
+package com.double_o.dambap.s3.presentation;
+
+import static com.double_o.dambap.s3.utils.MediaConstants.MEDIA_MAX_SIZE;
+
+import com.double_o.dambap.common.model.ResponseDto;
+import com.double_o.dambap.exception.dto.ErrorType;
+import com.double_o.dambap.exception.post.PostInvalidException;
+import com.double_o.dambap.exception.s3.S3InvalidException;
+import com.double_o.dambap.s3.application.S3Uploader;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.multipart.MultipartFile;
+
+@Slf4j
+@RestController
+@RequestMapping(path = "/api/images")
```

Comment:
```suggestion
@RequestMapping(path = "/api/v1/images")
```

버저닝을 남기면, 나중에 편해요! (버전업이 발생할때!)